src/views/UserSettings.js

```diff
@@ -0,0 +1,438 @@
+      sortDirections: ['descend', 'ascend'],
+      render: (role) => {
+        let color = stringToColor(role);
+        return (
+          <Tag
+            style={{ color: 'white', fontWeight: 'bold' }}
+            color={color}
+            key={role}
+          >
+            {role.toUpperCase()}
+          </Tag>
+        );
+      },
+    },
+    {
+      title: 'Total Members',
+      dataIndex: 'members',
+      key: 'members',
+      sorter: (a, b) => a.members.localeCompare(b.members),
+      sortDirections: ['descend', 'ascend'],
+    },
+    {
+      title: 'Action',
+      key: 'action',
+      render: (text, record) => (
+        <Button type="primary" danger href="#">
+          Leave
+        </Button>
+      ),
+    },
+  ];
+  return (
+    <Card id="users" style={{ marginBottom: '20px' }}>
+      <h1 style={{ fontSize: '2em', fontWeight: 'bold' }}>
+        Manage Organizations &nbsp;
+        <Tooltip
+          title={
+            <p style={{ textAlign: 'center' }}>
+              View your existing organization memberships and leave if you
+              choose
+            </p>
+          }
+          placement="right"
+        >
+          <QuestionCircleTwoTone style={{ fontSize: '0.8em' }} />{' '}
+        </Tooltip>
+      </h1>
+      <Search
+        style={{ width: '50%', marginBottom: '20px' }}
+        placeholder="User Search"
+        enterButton
+        onSearch={console.log}
+      />
+      <Table
+        columns={columns}
+        dataSource={organizations}
+        onChange={onChange}
+        pagination={{ pageSize: 10 }}
+        scroll={{ y: 240 }}
+      />
+    </Card>
+  );
+}
+
+function Sidebar() {
+  return (
+    <Affix offsetTop={60}>
+      <Sider width={250}>
+        <Menu
+          mode="inline"
+          theme="light"
+          defaultOpenKeys={['users', 'resources']}
+          style={{ height: '100%', borderRight: 0 }}
+        >
+          <Menu.Item
+            key="dashboard"
+            icon={<AreaChartOutlined />}
+            style={{ marginTop: '30px' }}
+            onClick={() => {
+              window.scrollTo({
+                top: 0,
+                behavior: 'smooth',
+              });
+            }}
+          >
+            User Overview
+          </Menu.Item>
+          <Menu.Item
+            key="organizations"
+            icon={<TeamOutlined />}
+            onClick={() => {
+              window.scrollTo({
+                top: 250,
+                behavior: 'smooth',
+              });
+            }}
+          >
+            Organizations
+          </Menu.Item>
+          <Menu.Item
+            key="pending"
+            icon={<FileProtectOutlined />}
+            onClick={() => {
+              window.scrollTo({
+                top: 600,
+                behavior: 'smooth',
+              });
+            }}
+          >
+            Uploaded Resources
+          </Menu.Item>
+        </Menu>
+      </Sider>
+    </Affix>
+  );
+}
```

Comment:
Would it make sense to make a generic sidebar component for this? Seems upper super similar to the admin settings one and would be nice reuse w/o so much copying.